src/test/fuzz/util.h

```diff
@@ -642,6 +664,47 @@ class FuzzedSock : public Sock
         return random_bytes.size();
     }
 
+    int Connect(const sockaddr*, socklen_t) const override
```

Comment:
Why? It is a bit shorter without the parameter names. Omitting parameter names if they are not used I think is a good practice because it makes it obvious that they are not used.

Also, specifying parameter names and not using those parameters in the function will produce a compiler warning with `-Wunused-parameter` (which is explicitly disabled in Bitcoin Core, I guess because of so many functions that violate it that it is hard to fix).

Reply:
Thanks for the explanation.